packages/manager/modules/billing/src/orders/purchaseOrders/billing-orders-purchases.service.js

```diff
@@ -98,7 +98,7 @@ export default class BillingOrdersPurchasesService {
   }
 
   minDateForEndDate() {
-    this.moment = moment();
+    this.moment = moment(null, 'YYYY-MM-DD');
     return this.moment.add(1, 'day').toDate();
```

Comment:
The both lines produce an `Invalid Date`, is't expected behaviour ?

Reply:
Indeed, in my tests it came up false. I was misled by the result. I made some changes.